lib/src/value/calculation.dart

```diff
@@ -120,6 +121,25 @@ class SassCalculation extends Value {
     return SassCalculation._("max", args);
   }
 
+  /// Creates a `sqrt()` calculation with the given [argument].
+  ///
+  /// Each argument must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation]. It must be passed at least one argument.
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value sqrt(Object argument) {
+    argument = _simplify(argument);
+    if (argument is! SassNumber) {
+      return SassCalculation.sqrt(argument);
+    }
+    argument.assertNoUnits();
+
+    return SassNumber(math.sqrt(argument.value));
```

Comment:
Once we've handled the case where at least one argument doesn't simplify to a number, we should probably call out to shared logic used both here and in the `sass:math` functions in `lib/src/functions/math.dart` to ensure the behavior stays consistent. The way to do that would be to create a new math library (probably in the `util` folder) that implements the behavior that's currently done directly in `lib/src/functions/math.dart` and then update both here and that library to call out to it.